tools/raisinbread_refresh.php

```diff
@@ -0,0 +1,138 @@
+<?php declare(strict_types=1);
+/**
+ * This script is intended for developers working with test data. It DROPs all
+ * LORIS core tables as well as "Raisinbread" test instrument tables.
+ * For this reason, it should obviously not be run on any server wil live data.
+ *
+ * After dropping the tables, it will source the Raisinbread test data using
+ * the commands found in raisinbread/README.md.
+ *
+ * The script also restores the url and host config settings to their pre-drop
+ * values. This prevents an issue where a developer will need to manually
+ * change these values in a MySQL shell when they are not hosting a LORIS
+ * on localhost.
+ *
+ * Finally, the script runs tools/resetpassword.php so that the default admin
+ * password with the Raisinbread data set is not used.
+ *
+ * In order to prevent accidental data loss, the script prompts the user to
+ * manually type the exact name of the database which will be affected. It
+ * also requires the user to manually enter their database password before
+ * dropping and importing.
+ *
+ * PHP Version 7
+ *
+ * @category Main
+ * @package  Loris
+ * @author   John Saigle <john.saigle@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris-Trunk/
+ */
+
+$info = <<<INFO
+This script is used by LORIS developers to empty their databases and replace
+them with new test data.
+DO NOT RUN THIS SCRIPT ON PRODUCTION.
```

Comment:
you should check if the sandbox flag is enabled and abort if it's not

Reply:
Done!